### PR TITLE
TASK: Fix testsuite for simple use cases (in phpunit)

### PR DESCRIPTION
**Pull request by mhsdesign**
Followup to https://github.com/neos/neos-development-collection/pull/5344 to remove behat specific logic from test fakes so they can be used in php unit tests easily.

**Upgrade instructions**

**Review instructions**

**Checklist**

- [ ] Code follows the PSR-2 coding style
- [ ] Tests have been created, run and adjusted as needed
- [ ] The PR is created against the [lowest maintained branch](https://www.neos.io/features/release-roadmap.html)
- [ ] Reviewer - PR Title is brief but complete and starts with `FEATURE|TASK|BUGFIX`
- [ ] Reviewer - The first section explains the change briefly for change-logs
- [ ] Reviewer - Breaking Changes are marked with `!!!` and have upgrade-instructions

### CODE REVIEW

File: Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml
Comment (bwaidelich) [inline Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml:1]: Why was this moved from Configuration/Testing/Behat/ to Configuration?

**Review by bwaidelich — COMMENTED** [review]
Thank you for your work on this. I can imagine, it wasn't joyful :)
It's not an easy pr to review (especially on a phone screen).
So just a mini comment and the question: shouldn't we replace all the `Gherkin...` fakes? They always made me wonder why we don't translate the input

File: Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml
Comment (mhsdesign) [reply Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml:1]: i think this comment is now outdated after fixing cleaning up the diff :)

**Comment by mhsdesign** [comment]
Ha thanks for the attempted early review. This was planned as a followup to https://github.com/neos/neos-development-collection/pull/5344 but i hadnt rebased so far. The diff is now way more precise.

File: Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml
Comment (bwaidelich) [reply Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml:1]: OK, I see that it wasn't moved with this PR – but I still wonder why it is located at `/Configuration/Settings.yaml` and not `/Configuration/Testing/Behat/Settings.yaml`

Comment (kitsunet) [reply Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml:1]: Good point, how does anythnig work wit hhte dev-collection

Comment (mhsdesign) [reply Neos.ContentRepository.BehavioralTests/Configuration/Settings.yaml:1]: thanks good catch will move it to `Testing` i messed that up when adjusting the para tests ^^

**Review by kitsunet — APPROVED** [review]
Go for merge
